src/plugins/data/common/search/aggs/buckets/lib/time_buckets/time_buckets.ts

```diff
@@ -172,38 +177,31 @@ export class TimeBuckets {
    * @param {object|string|moment.duration} input - see desc
    */
   setInterval(input: null | string | Record<string, any>) {
-    let interval = input;
-
-    // selection object -> val
-    if (isObject(input) && !moment.isDuration(input)) {
-      interval = input.val;
-    }
+    this._i = undefined;
+    this._originalInterval = null;
 
-    if (!interval || interval === autoInterval) {
-      this._i = autoInterval;
+    if (input === autoInterval) {
+      this._i = 'auto';
```

Comment:
use autoInterval variable

Reply:
we are no longer handling the case where input is null/undefined or where input is object (Record<string, any>) and the val on the object is a string date... is this intentional ?

Reply:
Thank you, my fail with `null/undefined`. But for `object` it should still work. I just moved it below. 
![image](https://user-images.githubusercontent.com/20072247/108469724-d68c1700-7299-11eb-9ee6-39a5c17941b5.png)